modules/k8s/create-ns.sh

```diff
@@ -1,15 +1,13 @@
 #!/bin/bash
 
 K3D_CLUSTER_NAME=${K3D_CLUSTER_NAME:-"local"}
-KIND_CLUSTER_NAME=${KIND_CLUSTER_NAME:-"kind"}
 
-if [ "$(k3d cluster list | grep -o -E "^${K3D_CLUSTER_NAME}")" ] || \
-   [ "$(kind get clusters | grep -o -E "^${KIND_CLUSTER_NAME}")" ]; then
+if [ "$(k3d cluster list | grep -o -E "^${K3D_CLUSTER_NAME}")" ]; then
   set +e
   namespaces=$(tk env list --json | jq -r '.[].spec.namespace' | sort | uniq)
   while IFS= read -r namespace; do
     echo "Creating namespace $namespace ....."
-    out=$(kubectl create namespace $namespace 2>&1)
+    out=$(kubectl --context k3d-local create namespace $namespace 2>&1)
```

Comment:
Avoid hardcoding `k3d-local`, it's derived from `${K3D_CLUSTER_NAME}` var I believe